src/controllers/profileInitialSetupController.js

```diff
@@ -275,7 +300,7 @@ const profileInitialSetupController = function (
           };
 
           res.send({ token }).status(200);
-
+          ProfileInitialSetupToken.update();
```

Comment:
No arguments were passed to the update function. Was it intended to update the 'isSetupCompleted' property?

Reply:
Correct. Will include the updated code in next commit.